### DISCUSSION TITLE
Does kyuubi ldap support configuring user whitelists？

### DISCUSSION THREAD

**Discussion by conorzhao** [apache/kyuubi#5393] [Q&A] [answered]
I would like to configure some users separately to skip ldap authentication。

**Comment by pan3793** [comment]
`kyuubi.authentication.ldap.userFilter` should work.

https://kyuubi.readthedocs.io/en/master/security/ldap.html
https://cwiki.apache.org/confluence/display/Hive/User+and+Group+Filter+Support+with+LDAP+Atn+Provider+in+HiveServer2#UserandGroupFilterSupportwithLDAPAtnProviderinHiveServer2-UserandGroupFilterSupportwithLDAP

**Reply by conorzhao** [reply]
I tested and found that this configuration is not a whitelist, but a search user list. After configuration, only users in the configuration list can authenticate and log in

**Reply by pan3793** [reply]
what do you mean "whitelist"?

**Reply by conorzhao** [reply]
Configured users can skip permission authentication

**Reply by pan3793** [reply, marked as answer]
sounds like an uncommon cases, consider implement a custom authz plugin

**Reply by conorzhao** [reply]
ok，thx
